database/__init__.py

```diff
@@ -0,0 +1,45 @@
+from pyramid.config import Configurator
+
```

Comment:
I'm not going to comment on coding style in this review but before this is pulled in we should clean up all of the additional blank lines and other spacing oddities.

Reply:
Removed unnecessary blank lines